src/app/employee-salary/employee-salary.component.html

```diff
@@ -0,0 +1,48 @@
+<div *ngIf="!fetchDone">
+  <div class="row data-container">
+    <mat-spinner [diameter]="40"></mat-spinner>
+  </div>
+</div>
+<ng-container *ngIf="fetchDone && errorMsg">
+  <div class="select-file-error">{{errorMsg}}</div>
+  <div class="navigate-desc">To navigate to home page <a routerLink="/home">Click here</a></div>
+</ng-container>
+<div class="row btn-container" *ngIf="fetchDone && !errorMsg">
+  <div class="col-3">
+    <button class="btn btn-primary back-btn" (click)="previousPage()">Back</button>
+  </div>
+  <div class="col-6 head-content">Salary Slip of Employees</div>
+  <div class="col-3 send-btn-container">
+    <button class="btn btn-primary delete-btn" [disabled]="checkedList.length == 0" (click)="deleteItem()">Delete</button>
```

Comment:
We can simply add a check on `checkedList && checkedList.length`. Don't need to check length ==0